web/packages/design/src/ThemeProvider/index.tsx

```diff
@@ -159,4 +47,19 @@ export function shouldForwardProp(propName: string, target: WebTarget) {
   return true;
 }
 
-export default ThemeProvider;
+/** Uses a theme from a prop. Can be used in tests, storybook or in an app. */
+export function StaticThemeProvider(props: {
```

Comment:
nit: maybe its just me but since this uses theme from a props, `Static` reads misleading to me. Do we also have a dynamic theme provider? if yes, it would be great to mention that here with a small note when to use one versus another.

Reply:
Yeah, maybe `Static` isn't the best term here. I renamed it to `ConfiguredThemeProvider` since under the hood it sets up some `styled-components` components.